src/components/comments/Question.js

```diff
@@ -1,9 +1,58 @@
-import React from "react";
-
+import React, { useState } from "react";
+import "../mainContent/mainContent.css";
+import "./comments.css";
+import Answer from "./Answer.js";
 export default function Question(props) {
+
+  /* the index of the current question inside the followups array */
+  const questionIndex = props.followups.findIndex(function(item, i){
+    return item.question === props.questionText;
+  });
+
+  /* keeps track of whether or not the answers are hidden*/
+  const [hidden, setHidden] = useState(true); 
+
+  const hiddenHandler = () => {
+    setHidden(!hidden);
+  };
+
+  /* renders all of the answers associated with the question */
+  const answerList = () => {
+    let answers = props.followups.filter((s, i) => i > questionIndex && i <= questionIndex + props.numAnswers).map((a, index) => {
+      console.log(questionIndex);
+      return (
+        <Answer
+          key={index}
+          answerText={a.answer}
+          pic={props.followups[0]} /* passes profilePic to Answer */
+        />
+      );
+    })
+
+    return answers;
+  };
+
   return (
-    <div className="question comment-bubble body-text text-bold">
-      {props.questionText}
+    <div>
+      <div className="single-comment-container">
+        <img
+          className="comment-profile-pic"
+          src={props.pic.profilePic}
+          alt="An icon that matches up with the post"
+        />
+        <div className="comment-bubble body-text">
+          <div><b>{props.questionHead}</b></div>
```

Comment:
Similarly, I don't think we should use `b` here for bold text - instead, let's abstract that away to a CSS class!